test/src/e2e_vm_tests/test_programs/should_fail/illegal_continue/Forc.lock

```diff
@@ -1,3 +0,0 @@
-[[package]]
```

Comment:
Is the removal of this file intentional?

Reply:
No! I have restored the lock file. TY